### PR TITLE
Signature and IRS redux

### LINKED ISSUE #210: Update IRS and Signature to handle POST
#208 got these components most of the way, but we also need to be able to handle the `POST` for each when a user checks/un-checks the checkbox.

### PR DESCRIPTION
**Pull request by awolfe76**
Closes #210. Progress toward #180.
- implements redux on the signature and irs components
  - both GET and POST are handled
- updated the API and responses (see `server/json/irs.json` and `server/json/receipt.json`)
- also implements a new way to handle the status of the application

### CODE REVIEW

**Comment by awolfe76** [comment]
@wpears - tests are passing!

File: src/js/api.js [hunk 1]
```diff
@@ -54,27 +55,27 @@ export function getLatestSubmission(){
 }
 
 export function getEditsByType(submission){
-  return sendFetch(`/submissions/${submission}/edits`)
+  return sendFetch('/submissions/${submission}/edits')
```
Comment (wpears) [inline src/js/api.js]: These should be the original grave marks because we're using ES2015 string substitution here

Comment (wpears) [reply src/js/api.js]: (applies throughout the file)

**Review by wpears — CHANGES_REQUESTED** [review]
Everything looks good, but the api grave stuff will need to be reverted before merge

File: src/js/api.js (hunk 1 above)
Comment (awolfe76) [reply src/js/api.js]: 👍  - ready to go.

**Pull request merged**